### ISSUE TITLE
iDummyCursor Review

### ISSUE THREAD

**Issue by AlexFine** [benjaminsage/DummyCursor#1]
TLDR: Initially confusing but figured it out. Most comments here are related to ambiguity of use. 

1. It was definitely initially confusing what this package did. When I ran the package I did not know what to expect. I realized it's a classic cursor but that yielded some questions. 
A) Why is it called DummyCursor & not CustomCursor? What is the "Dummy" for? 
B) Do I incorporate this package in a textfield? How? 
C) Is there any way to incorporate this package not in a textfield? If not, why is it a standalone element & not incorporated into a textfield of some kind? 
D) I had a strong reaction that this package should seamlessly fit into our `TextView` but wasn't exactly sure how. 

2. Confusion as to "what's next." A custom cursor is cool, but I didn't really know what I could do with that. For example, what does a really cool custom cursor look like? So my reaction was to first look towards the inits, and then the modifiers to see what the package suggested. This just yielded color. So then I was a little confused – what other modifications are there? & why isn't easy to make those modifications? 

3. I was also confused how to modify the package. The ReadMe says it supports all standard view modifiers. So I tried setting a different width. And maybe I did that wrong but it didn't work. And because there was only one modifier, I wasn't sure what other modifiers I was supposed to use. 
<img width="398" alt="Screen Shot 2020-10-06 at 2 15 27 PM" src="https://user-images.githubusercontent.com/5194671/95261016-80cd3b80-07de-11eb-8ab6-a1fd122009a6.png">

So in conclusion, no bugs or anything. Cool idea & cool concept. Just a few small things that made implementation confusing tbh. Happy to talk through explicit changes to address some of these confusions.
